models/fetchTopics.model.js

```diff
@@ -0,0 +1,7 @@
+const db = require("../db/connection.js");
+
+function fetchTopics() {
+  return db.query(`SELECT * FROM topics`);
```

Comment:
Small note - it's best to take your info off the rows key in the model rather than the controller, as it's manipulating the data into the form you want which is the model's domain